vars/multicloudDeploy.groovy

```diff
@@ -22,25 +22,22 @@ import static com.sap.piper.Prerequisites.checkScript
 @Field Set STEP_CONFIG_KEYS = []
 
 @Field Set PARAMETER_KEYS = GENERAL_CONFIG_KEYS.plus([
-    /** The stage name. If the stage name is not provided, it will be taken from the environment variable 'STAGE_NAME'.*/
-    'stage',
```

Comment:
It is removed because the staging is done per step: neoDeploy and cloudFoundryDeploy